### PR TITLE
Fix Plot Legend placement

### PR DESCRIPTION
**Pull request by charlesh88**
### Overview
Addresses https://github.com/nasa/openmct/issues/2486. Legend position on left and right of overlay plots now maintains position when expanded:

Screen Shot 2019-10-30 at 4 43 18 PM
Screen Shot 2019-10-30 at 4 43 27 PM

### Author Checklist
1. Changes address original issue? Y
1. Unit tests included and/or updated with changes? N/A
1. Command line build passes? Y
1. Changes have been smoke-tested? Y

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0, shrink-to-fit=no">
         <meta name="apple-mobile-web-app-capable" content="yes">
-        <title></title>
+        <title>test</title>
```
Comment (deeptailor) [inline index.html]: Is this left here by mistake?

Comment (charlesh88) [reply index.html]: Yep. Good catch, thanks. Fixed and committed.

**Comment by charlesh88** [comment]
Testing notes:
1. Create a new Overlay Plot and drag some telemetry into it.
1. In the Inspector Legend properties, set the Position to "Left".
1. Observe: the Legend is placed to the left of the plot area. 
1. Expand the Legend and observe: the expanded legend should remain on the left, and the plot area should collapse somewhat to the right.
1. In the Inspector Legend properties, change the Position to "Right".
1. Repeat steps 3 and 4 with expectation that the legend will be on the right.
1.  In the Inspector Legend properties, change the Position to "Top".
1. Repeat steps 3 and 4 with expectation that the legend will be on the top.

Extra credit: save the Overlay plot, place it in a Layout, and make it small. Perform test steps and ensure the view behaves as expected.

**Comment by charlesh88** [comment]
Testathon 12-06-19: verified fixed.

**Pull request merged**